packages/flutter/lib/src/material/date_picker.dart

```diff
@@ -370,6 +388,20 @@ class DatePickerDialog extends StatefulWidget {
   /// `initialEntryMode` parameter the next time the date picker is shown.
   final ValueChanged<DatePickerEntryMode>? onDatePickerModeChange;
 
+  /// An optional [switchToInputEntryModeIcon] argument can be used to
```

Comment:
Rather than copy/pasting the same text wherever it belongs, please use `@template` to define the text and `@macro` to copy it. You'll find examples in this file.

Reply:
I've added the templates and macros. 👍✅